test/index.js

```diff
@@ -36,7 +36,7 @@ test('\nfoo with .gitignore and no .npmignore', function (t) {
     });
 })
 
-test('bar with prepublish script', function (t) {
+test('@scope/bar with prepublish script', function (t) {
```

Comment:
@chinesedfan Instead of changing this test, can you copy it to a new test (with its own fixture)?

Reply:
Yeah totally agreed .. DON"T change any existing tests please, otherwise this isn't acceptable.

I overlooked that, @vweevers thanks for catching it.
Please don't merge as is.

Reply:
My fault and fixed.